File: sorts/bubble.py
#UDF
def bubblesort(nums):
    for i in range(len(nums) - 1, 0, -1):
        for j in range(i):
            if nums[j] > nums[j + 1]:
                temp = nums[j]
                nums[j] = nums[j + 1]
                nums[j + 1] = temp
    return nums

#Unsorted List
nums = [9, 3, 7 ,4, 6 , 2, 1, 5, 8, 0]

#Function Call
num = bubblesort(nums)
print(num)

#Another way

list = eval(input("\n\nEnter a list of elements: "))
n = len(list)
for i in range(n):
    for j in range(0, n - i - 1):
        if list[j] > list[j + 1]:
            list[j], list[j + 1] = list[j + 1], list[j]
print("List after sorting : ", list)

File: sorting in c/bubble.c
#include<stdio.h>

int main()
{
    int a[5] = {5, 3, 1, 2, 4, 8, 6, 7, 9, 10};
    
    for(int i = 0; i < 5; i++)
    {
        for(int j = 0; j < i + 1; j++)
        {
            if(a[i] < a[j])
            {
                int temp = a[i];
                a[i] = a[j];
                a[j] = temp;
            }
        }
    }
    
    for(int i = 0; i < 5; i++)
    {
        printf("%d", a[i]);
    }

    return 0;
}